config/settings.yml

```diff
@@ -113,6 +113,8 @@ global_navigation:
   pages:
     create_case:
       path: /cases/new
+
```

Comment:
Probably don't need this whitespace, maybe remove?

Reply:
yes, removed, thanks